query_execution.go

```diff
@@ -520,7 +520,7 @@ func mergeExecutionResultsRec(src interface{}, dst interface{}, insertionPoint [
 					}
 					if srcID == dstID {
 						for k, v := range result {
-							if k == "_id" || k == "id" {
+							if k == "_bramble_id" {
```

Comment:
Then here's where https://github.com/movio/bramble/issues/93 was breaking... this special case for the `id` field prevented it from resolving standard GraphQL aliases. Now that everything revolves around `_bramble_id`, we can leave `id` alone an allow it to resolve as a normal field.